recommendation_system/candidate_layer/candidate_model.py

```diff
@@ -0,0 +1,55 @@
+
+
+from typing import Dict, List
+from recommendation_system.candidate_layer.base import BaseCandidateModel
+from candidate_model_util import FastBM25
+
+
+# demo data
+from organic_data import user_features, article_data
```

Comment:
it's a good idea and maybe you can share these fixtures with other DS.
c.c. [hane1818](https://github.com/hane1818) [PuChenTW](https://github.com/PuChenTW)